openadapt/start.py

```diff
@@ -1,5 +1,4 @@
-"""
-Implements the code needed to update the OpenAdapt app if needed.
+"""Implements the code needed to update the OpenAdapt app if needed.
```

Comment:
Upon further reflection, I think this would be better worded as, "Implements the code necessary to update the OpenAdapt app if required"

Reply:
Addressed in https://github.com/OpenAdaptAI/OpenAdapt/pull/289/commits/d88767ff785961d7eb36ca589d96f707a019b507